diagnostic.md

```diff
@@ -7,36 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+We navigate in terminal ot the folder where we want to store our local copy of
+the directory then type:
+git clone git@github.com:treep78/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+We must first move into the repository with:
+cd git-basics-diagnostic
```

Comment:
this is not technically incorrect, but we were looking for "create a new branch"